nemo/collections/nlp/models/nlp_model.py

```diff
@@ -249,7 +249,9 @@ def load_from_checkpoint(
                 config_kwargs.pop('trainer')
             checkpoint[cls.CHECKPOINT_HYPER_PARAMS_KEY].update(config_kwargs)
 
-            model = cls._load_model_state(checkpoint, strict=strict, **kwargs)
+            model = cls._load_model_state(
+                checkpoint, strict=strict, cfg=checkpoint[cls.CHECKPOINT_HYPER_PARAMS_KEY], **kwargs
```

Comment:
This PR needs to be merged first - https://github.com/NVIDIA/NeMo/pull/3172

This line is using a deprecated thing which breaks PTL for a notebook.

Reply:
Okay I added @fayejf 's change.